Documentation/Getting-Started/quickstart.md

```diff
@@ -36,7 +36,7 @@ In order to configure the Ceph storage cluster, at least one of these local stor
 A simple Rook cluster can be created with the following kubectl commands and [example manifests](https://github.com/rook/rook/blob/master/deploy/examples).
 
 ```console
-$ git clone --single-branch --branch master https://github.com/rook/rook.git
+$ git clone --single-branch --branch v1.10.0-beta.0 https://github.com/rook/rook.git
```

Comment:
We need to keep the master tags in a few places like this that were changed in the release branch. 
```suggestion
$ git clone --single-branch --branch master https://github.com/rook/rook.git
```

Reply:
I actually decided to keep these on `master` in this PR. I'll handle conflicts on the backport.